client/client-multi/client-c/listfiles/platform/x86-64/CMakeLists.txt

```diff
@@ -14,24 +14,22 @@
 # limitations under the License.
 #
 
-find_package(OpenSSL REQUIRED)
-
-set(KAA_SOURCE_FILES 
+set(KAA_SOURCE_FILES
         ${KAA_SOURCE_FILES}
         ${KAA_SRC_FOLDER}/platform-impl/posix/kaa_client.c
-        ${KAA_SRC_FOLDER}/platform-impl/posix/sha.c
         ${KAA_SRC_FOLDER}/platform-impl/posix/logger.c
         ${KAA_SRC_FOLDER}/platform-impl/posix/file_utils.c
-        ${KAA_SRC_FOLDER}/platform-impl/posix/key_utils.c
         ${KAA_SRC_FOLDER}/platform-impl/posix/status.c
         ${KAA_SRC_FOLDER}/platform-impl/posix/configuration_persistence.c
+        ${KAA_SRC_FOLDER}/platform-impl/common/sha.c
+        ${KAA_SRC_FOLDER}/platform-impl/common/common_key_utils.c
```

Comment:
It is called common just because some of the platforms share it. Not all of the platforms use it.